# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../enterminus/init.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- enterminus/src/bin/main.c_orig	2006-02-19 12:57:15.000000000 +0100
+++ enterminus/src/bin/main.c	2006-02-19 13:22:09.000000000 +0100
@@ -12,6 +12,10 @@
    Evas        *evas;   /* evas       */
    Evas_Object *term;   /* terminal   */
 
+   evas_init();
+   ecore_init();
+   ecore_evas_init();
+
    ee = ecore_evas_software_x11_new(0, 0, 0, 0, 0, 0);
    ecore_evas_show(ee);
    evas = ecore_evas_get(ee);
